src/screens/SUE/SueReportScreen.tsx

```diff
@@ -219,14 +219,25 @@ export const SueReportScreen = ({
   }, []);
 
   const storeReportValues = async () => {
-    await addToStore(SUE_REPORT_VALUES, { selectedPosition, serviceCode, ...getValues() });
+    await addToStore(SUE_REPORT_VALUES, {
+      currentProgress: currentProgress + 1,
+      selectedPosition,
+      serviceCode,
+      ...getValues()
+    });
   };
 
   const readReportValuesFromStore = async () => {
     const storedValues = await readFromStore(SUE_REPORT_VALUES);
 
     if (storedValues) {
       setStoredValues(storedValues);
+      setCurrentProgress(storedValues.currentProgress);
+      scrollViewRef?.current?.scrollTo({
+        x: device.width * storedValues.currentProgress,
+        y: 0,
+        animated: true
+      });
```

Comment:
somehow this does not work efficiently, here two cases, where i start at one step less then i left before:

|||
|---|---|
|https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/0556ad18-3a85-4867-891c-83581c3fc7dc|https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/cfb1c770-0cef-4637-b7e2-3a17ecfa9176|

Reply:
I have searched for the problem, but I can't solve it exactly. I can describe the problem as follows:

Open application -> Go to ReportScreen -> Select category -> Click on Weiter button (data is saved) -> Go back to HomeScreen -> Go to ReportScreen again -> ReportScreen starts from step 2

There is no problem here. ScrollTo works correctly. But after that currentProgress is not saved correctly in AsyncStorage.

Write title -> Press Weiter button (data is saved) -> Return to HomeScreen -> Go back to ReportScreen -> ReportScreen starts from step 2 (expected to start from step 3)

currentProgress cannot be correctly saved to AsyncStorage. I am investigating why this might be happening.

Reply:
we need to invest more tim ein this problem. for now, please remove it in this part of the project and keep it in a separate branch for later.
for v1 we will always start again from step 1